app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/Answer.kt

```diff
@@ -1,16 +1,36 @@
 package org.simple.clinic.teleconsultlog.teleconsultrecord
 
+import android.os.Parcelable
 import androidx.room.TypeConverter
 import com.squareup.moshi.FromJson
 import com.squareup.moshi.ToJson
+import kotlinx.android.parcel.Parcelize
 import org.simple.clinic.util.room.SafeEnumTypeAdapter
 
-sealed class Answer {
+sealed class Answer : Parcelable {
 
+  companion object {
+
+    fun fromBoolean(boolean: Boolean): Answer {
+      return if (boolean) Yes else No
+    }
+
+    fun toBoolean(answer: Answer): Boolean {
+      return when (answer) {
+        Yes -> true
+        No,
+        is Unknown -> false
+      }
+    }
```

Comment:
I would like to avoid making boolean functions like this a part of the domain since they hide a lot of information. Can we move these to the screen where they are being used so that the potential for misuse does not spread?